go/internal/initutil/manager.go

```diff
@@ -188,8 +200,8 @@ func (m *Manager) GetContext() context.Context {
 
 func (m *Manager) RunWorkers() error {
 	m.workers.Add(func() error {
```

Comment:
missing a check for init timeout in mains that only call RunWorkers without getting anything first